pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -30,7 +30,11 @@ stdenv.mkDerivation {
     rm -rv ofl/comfortaa/*.ttf \
       ofl/mavenpro/*.ttf \
       ofl/muli/*.ttf \
-      ofl/oswald/*.ttf
+      ofl/oswald/*.ttf \
```

Comment:
This `\` is pointless, it cannot join lines through multiple new lines or comments.

Reply:
You're right, I didn't realize I had left that in there.